MANIFEST.in

```
@@ -1,10 +1,92 @@
-prune .github
-prune requirements
+# A generic template for "MANIFEST.in" files in Scitools repos.
+# NOTE: most of this can't be duplicated as-is into any given repo, but suggests
+#     guidelines + structure.
+# For the syntax of "MANIFEST.in" files, see:
+#     https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#using-manifest-in
+
+# General principles
+#     - enable user to build locally, as well as in CI
+#     - make it ignore temporary files generated by dev processes, e.g. coverage checks
+#     - encode typical decisions, e.g. whether we package docs, requirements etc
+
+#---------
+# SECTION: main code sources
+#
 recursive-include src *.py *.pyx
```

If the policy is full explicitness, then there are a couple more here:

```suggestion
recursive-include src *.c *.pp *.py *.pyx *.so
```

The PP file is part of the tests.

The other two don't show up when viewing the repo, but do appear during the build process. They are associated with the C layer.